### DISCUSSION TITLE
Where should I run the prisma migrate deploy command, if the database cannot be directly accessed through GitHub Action

### DISCUSSION THREAD

**Discussion by Lantianyou** [prisma/prisma#22712] [Q&A]
### Question

Prisma's doc highly recommending runing migrate deploy in GitHub Action, however, I can only access the production db through a jump machine via a private key stored in my local machine. I believe such a case is pretty normal since database normally do not expose public IP. Then where should I run prisma migrate deploy. It would be best if anyone can help

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @Lantianyou 👋 

Thank you for raising this question. You would need to:

1. Configure GitHub Actions to SSH into the bastion host first.
2. From the bastion host, forward SSH to the production database server.
3. Execute `prisma migrate deploy` within this SSH session.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
